### DISCUSSION TITLE
Is there a way to configure multiple filters for one component?

### DISCUSSION THREAD

**Discussion by cj19** [quarkusio/quarkus#42312] [Q&A]
I am aware of that there is a configuration parameter which allows to set up filter, but it only accepts one String (quarkus.log.console.filter). How can I define multiple filters for one app?

[a comment by an automated account is not shown]

**Comment by radcortez** [comment]
I don't know if that is even possible at the API level. A `Handler` only takes a single `Filter`: https://docs.oracle.com/en%2Fjava%2Fjavase%2F17%2Fdocs%2Fapi%2F%2F/java.logging/java/util/logging/Handler.html#setFilter(java.util.logging.Filter).

Could you tell me what you're trying to achieve?
